### ISSUE TITLE
No error message when CSS parsing fail

### ISSUE THREAD

**Issue by vjeux** [NativeScript/NativeScript#30]
I used the wrong way to comment in CSS. Unfortunately, there was no error, no stack trace, no file/line number. It successfully built, start to open the app and insta crashed. Would be nice to have an error message saying why.

```css
.message {
    font-size: 20px;
    color: #284848;
//    horizontal-align: center;
}
```

```
** BUILD SUCCEEDED **

Project successfully built
Using  /Users/vjeux/random/hello-world/platforms/ios/build/emulator/hello-world.app
Cleaning up before starting the iOS Simulator
Starting iOS Simulator
Specify the timeout in number of seconds to wait. It should be greater than 0. Default value 90 seconds will be used.
Session started without errors.
```

**Comment by vakrilov** [comment]
The app should not crash if the CSS is invalid. It is probably a bug we fixed as I wasn't able to reproduce it with the latest bits.
That said - we will consider adding some build-time check of the XML and CSS files. For now you can check for errors on in the error trace log. Here is how to enable it:
```
var trace = require("trace");
trace.setCategories(trace.categories.Error);
trace.enable();
```
